api/cloudprotocol/cloudprotocol.go

```diff
@@ -95,9 +95,10 @@ const (
 
 // Service instance states.
 const (
-	InstanceStateActive   = "active"
-	InstanceStateFailed   = "failed"
-	InstanceStateInactive = "inactive"
+	InstanceStateActive     = "active"
```

Comment:
Put it in logical order: activating, active, inactive, failed

Reply:
Done